terraform_files/variables-libvirt.tf

```diff
@@ -59,22 +79,22 @@ variable "libvirt_network_mtu" {
 }
 
 variable "libvirt_master_ips" {
-  type        = list(string)
+  type        = list(list(string))
```

Comment:
those changes will break e2e tests, they use the same files.

Reply:
E2e where? Where's the code? I made sure to change all code that references this variable.

Reply:
try running e2e tests. Check that they pass